app/views/catalog/_show.html.erb

```diff
@@ -1,6 +1,6 @@
 <% doc_presenter = show_presenter(document) %>
 <%# default partial to display solr document fields in catalog show view -%>
-<dl class="dl-horizontal  dl-invert ddtest">
+<dl class="document-metadata dl-invert ddtest row">
```

Comment:
this cleans up the data display quite a bit - does it mess up harvard? If so, we can put it back and I'll override the view partial

Reply:
@summer-cook do you have any insight on this?

Reply:
CURIOSity actually [already has this change locally](https://gitlab.com/notch8/harvard-curiosity/-/blob/main/app/views/catalog/_show.html.erb#L8), so this doesn't make a huge difference there either way. I also got confirmation from Enrique that it shouldn't mess up any other Harvard apps that use this gem.